### PR TITLE
Redis Configuration

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- Redis deployment and service

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj @avikganguly01 @somanath21

File: helm/ph-ee-engine/templates/dev-redis.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{{- if .Values.redis.enabled -}}
```
Comment (fynmanoj) [inline helm/ph-ee-engine/templates/dev-redis.yaml:1]: Do we need to add the redis chart like this? Do t we have any ready made chart available from online providers like bitnami?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Clarification Requested

**Comment by fynmanoj** [comment]
closing this PR as we need to add the redis as a depoendency

**Pull request closed without merging**